pkgs/applications/misc/calibre/default.nix

```diff
@@ -196,9 +196,8 @@ stdenv.mkDerivation (finalAttrs: {
       free and open source and great for both casual users and computer experts.
     '';
     changelog = "https://github.com/kovidgoyal/calibre/releases/tag/v${finalAttrs.version}";
-    license = if unrarSupport
-              then lib.licenses.unrar
-              else lib.licenses.gpl3Plus;
+    license = [ lib.licenses.gpl3Plus ]
+      ++ lib.optional unrarSupport lib.licenses.unrar;
```

Comment:
Note that even though `unrar` should probably be `redistributable = true;`, this should be `[ lib.licenses.unrar lib.licenses.unfree ]` because GPL-3.0-or-later and the UnRAR licence are incompatible and cannot be legally combined or redistributed together.

Reply:
Relevant: https://github.com/NixOS/nixpkgs/issues/283706

Reply:
Like this?

```suggestion
      # the extra unfree is because gpl license cannot legally be combined with unfree licenses
      ++ lib.optionals unrarSupport [ lib.licenses.unrar lib.licenses.unfree];
```